integrations/langfuse/tests/test_tracing.py

```diff
@@ -19,6 +19,36 @@
 os.environ["HAYSTACK_CONTENT_TRACING_ENABLED"] = "true"
 
 
+@pytest.fixture
+def pipeline_with_env_vars(llm_class, expected_trace):
+    """Pipeline factory using environment variables for Langfuse authentication"""
```

Comment:
We should set 
```
monkeypatch.setenv("LANGFUSE_SECRET_KEY", "secret")
monkeypatch.setenv("LANGFUSE_PUBLIC_KEY", "public")
```
here to make sure the environment variables are set for the test.

Reply:
Yeah but we need real env vars here because these fixtures are used in integrations test that checks for existence of a trace afterwards

Reply:
You're right. My mistake.